src/index.js

```diff
@@ -108,6 +108,7 @@ const args = (() => {
         'xr',
         'size',
         'replace',
+        'onbeforeload'
```

Comment:
`onbeforeload` or `onBeforeLoad`? Below it's `onBeforeLoad`.

Personally I prefer `onbeforeload`.